src/org/openstreetmap/josm/plugins/mbtiles/MbtilesTileLoaderFactory.java

```diff
@@ -16,8 +16,7 @@ public MbtilesTileLoaderFactory(Connection connection) {
     }
 
     @Override
-    public TileLoader makeTileLoader(TileLoaderListener listener, Map<String, String> headers) {
+    public TileLoader makeTileLoader(TileLoaderListener listener, Map<String, String> headers, long minimumExpiryTime) {
```

Comment:
@iandees the new parameter `minimumExpiryTime` is currently ignored, optimally it should be respected in the tile loader factory.

Reply:
Do you know what it represents?

Reply:
As far as I understand, it represents the minimum amount of time for which tiles should remain in the cache (even if the server says the expiry time should be shorter).
See https://josm.openstreetmap.de/ticket/16249 for the development discussion about this change.